.github/actions/setup-tracing-windows/action.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Setup Tracing'
+description: 'Composite action to set up the tracing for test jobs'
+
+runs:
+  using: composite
+  steps:
+    - run: |
+        Invoke-WebRequest `
+          -Uri "https://github.com/jaegertracing/jaeger/releases/download/v1.50.0/jaeger-1.50.0-windows-amd64.tar.gz" `
+          -OutFile ".\jaeger-1.50.0-windows-amd64.tar.gz"
+        tar -zxvf ".\jaeger-1.50.0-windows-amd64.tar.gz"
+        Start-Process '.\jaeger-1.50.0-windows-amd64\jaeger-all-in-one.exe'
+        echo "OTEL_EXPORTER_OTLP_ENDPOINT=http://127.0.0.1:4318" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
+      shell: pwsh
```

Comment:
Oh that's good to know. Unfortunately GH was unhappy with this file and I'm not sure it will work better if I move this step into the `setup-tracing` action. See this run: https://github.com/moby/moby/actions/runs/6545687164/job/17775109401#step:3:1. Any ideas?

Reply:
Hum maybe the name should be unique? Can you try with `name: 'Setup Tracing Windows'`